docs/io/unified.rst

```diff
@@ -279,28 +279,26 @@ Time
 ~~~~
 
 By default, a `~astropy.time.Time` mixin column within a `~astropy.table.Table`
-or `~astropy.table.QTable` will be written to FITS as a standard Column using
-the ``format`` of the object (similar to the output when printing to the
-console).  For instance, if the ``format`` is ``iso`` then the FITS column will
-be a character string column.  None of the special header keywords associated
-with the Time FITS standard will be set.  When reading this back into Astropy,
-the column will be an ordinary Column instead of a `~astropy.time.Time` object.
-See the `Details`_ section below for an example.
+or `~astropy.table.QTable` will be written to FITS in full precision. This will be
+done using the FITS Time standard by setting the necessary FITS header keywords.
+
+The default read behaviour of a FITS table into an Astropy Table is to read the FITS
```

Comment:
Maybe rephrase a bit:
```
The default behaviour for reading a FITS table into an `~astropy.table.Table`
has historically been to convert all FITS columns to `~astropy.table.Column`
objects, which have closely matching properties. For some columns, however,
closer native astropy representations are possible, and one can indicate these
should be used by passing ``astropy_native=True`` (for backwards compatibility,
this is not done by default). This will convert columns conforming to the
FITS time standard to `~astropy.time.Time` instances, avoiding any loss of
precision. For example::
```